src/CMakeLists.txt

```
@@ -91,6 +91,10 @@ option(PYCSDIFF_PYTHON2 "Set to ON to build pycsdiff for Python 2" OFF)
 option(PYCSDIFF_PYTHON3 "Set to ON to build pycsdiff for Python 3" ON)
 
 macro(build_pycsdiff version)
+    # make FindPython use system Python 3 version
+    # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8287
+    set(Python3_FIND_UNVERSIONED_NAMES FIRST)
```

Make the variable generic because `FindPython2` supports `Python2_FIND_UNVERSIONED_NAMES` as well:
```suggestion
    # make FindPython${version} use system Python version
    # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8287
    set(Python${version}_FIND_UNVERSIONED_NAMES FIRST)
```

---

Addressed in 133076526aa324eee6748d272a6c39d110312b7f.